integration-tests/redis_test.go

```diff
@@ -8,15 +8,19 @@ import (
 )
 
 const (
-	redisServiceID                  = "e9c11b1b-0caa-45c9-b9b2-592939c9a5a6"
-	redisServiceName                = "csb-aws-redis"
-	redisServiceDescription         = "Beta - CSB Amazon ElastiCache for Redis - multinode with automatic failover"
-	redisServiceDisplayName         = "CSB Amazon ElastiCache for Redis (Beta)"
-	redisServiceDocumentationURL    = "https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.2/csb-aws/GUID-reference-aws-redis.html"
-	redisServiceSupportURL          = "https://aws.amazon.com/redis/"
-	redisServiceProviderDisplayName = "VMware"
-	redisCustomPlanName             = "custom-sample"
-	redisCustomPlanID               = "c7f64994-a1d9-4e1f-9491-9d8e56bbf146"
+	redisServiceID                        = "e9c11b1b-0caa-45c9-b9b2-592939c9a5a6"
+	redisServiceName                      = "csb-aws-redis"
+	redisServiceDescription               = "Beta - CSB Amazon ElastiCache for Redis - multinode with automatic failover"
+	redisServiceDisplayName               = "CSB Amazon ElastiCache for Redis (Beta)"
+	redisServiceDocumentationURL          = "https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.2/csb-aws/GUID-reference-aws-redis.html"
+	redisServiceSupportURL                = "https://aws.amazon.com/redis/"
+	redisServiceProviderDisplayName       = "VMware"
+	redisCustomPlanName                   = "custom-sample"
+	redisCustomPlanID                     = "c7f64994-a1d9-4e1f-9491-9d8e56bbf146"
+	deprecatedCacheSizePlanName           = "deprecated-cachesize-sample"
```

Comment:
nitpick: Maybe by introducing a new line between `redisCustomPlanID` and `deprecatedCacheSizePlanName` go formatter wouldn't try to reformat untouched lines to preserve alignment.

Having git/Github diffs only highlight added/changed lines is a big help when reviewing the code.
However, feel free to leave it as it is. But maybe keep it in mind for future commits.

Reply:
Also, I see you have done some cleanup of the code from cache_size.
I would recommend adding these changes in a different PR to keep this PR focused on the changes needed to make data_tiering work.
Also, feel free to be more strict when reviewing my PRs.
I wouldn't be bothered by you asking me to modify my PR to use less hardcoded strings in tests.

Reply:
I believe this PR is small enough that it is ok to leave the cleanup changes in it.

Reply:
https://github.blog/2018-05-01-ignore-white-space-in-code-review/